### DISCUSSION TITLE
Heal network - how long?

### DISCUSSION THREAD

**Discussion by drjjr2** [zwave-js/zwave-js-ui#962] [Q&A]
When I attempt to heal my network it gets as far as "Healing process IN PROGRESS. Healed 2 nodes" and then seems to stop. The debug window does not show any activity. I have the log level set to Debug.

**Comment by robertsLando** [comment]
@AlCalzone ?

**Reply by AlCalzone** [reply]
It can take several seconds up to a few minutes per node and you have to wake up battery-powered nodes to heal them. I'm not sure what exactly zwavejs2mqtt displays during the heal. It makes sense to show which nodes are skipped/healed/waiting.

**Reply by robertsLando** [reply]
It should be shown in controller status as it progress

**Reply by drjjr2** [reply]
What's the timeout for a sleeping node? I will say that the next node in line is an ECOLINK door/window sensor. But even when I open the door it still says "Asleep" in the status.

**Reply by AlCalzone** [reply]
There's no timeout. And opening the door will most likely not wake up the sensor. Check your manual how to do that (probably involves taking it apart and pressing some button.

**Reply by drjjr2** [reply]
OK, did that, they show "Awake" and I'm still stuck at "Healed 2 nodes"
Is there a log I can view to help figure this out?
